Doc/library/unittest.rst

```diff
@@ -531,16 +532,20 @@ Basic skipping looks like this::
            # windows specific testing code
            pass
 
+       def test_skip_me(self):
```

Comment:
If you take my second suggestion for the body of this example method, it could use a better name ("test_maybe_skipped" or similar, perhaps).